pkgs/by-name/ot/otree/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, stdenv
+, git
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "otree";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "fioncat";
+    repo = "otree";
+    rev = "v${version}";
+    hash = "sha256-ukKlypGY0+xsci/loAnf0Sz5K+82gVzkb+KnZ139cIA=";
+    leaveDotGit = true; # needed by build.rs
+  };
+
+  nativeBuildInputs = [ git ];
+
+  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdks.frameworks.IOKit;
```

Maybe add a blank line between `buildInputs` and `cargoLock`? Apologies for not including in my previous suggestion.
```suggestion
  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdks.frameworks.IOKit;
  
```

---

fine, i'll squash all commits in one commit tho, do you wanna add your self as a maintainer?

---

Thanks for the offer, @0x61nas, much appreciated, but I'll decline at this point. I might reconsider in the future…